### PR TITLE
Add function to send email and tests

### PR DESCRIPTION
**Pull request by kumaranvpl**

### CODE REVIEW

File: openai_agent/send_email.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import http.client
```
Comment (rjambrecic) [inline openai_agent/send_email.py:1]: Can you move this file into captn/email/send_email.py

File: openai_agent/send_email.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+from codecs import encode
+from os import environ
+from typing import Any, Dict
+
+INFOBIP_API_KEY = environ["INFOBIP_API_KEY"]
+INFOBIP_BASE_URL = environ["INFOBIP_BASE_URL"]
+
+
+def send_email(
+    *, from_email: str = "info@airt.ai", to_email: str, subject: str, body_text: str
```
Comment (rjambrecic) [inline openai_agent/send_email.py:12]: Please check with Harish which email address should we use.

info@captn.ai ?

File: openai_agent/send_email.py (hunk 1 above)
Comment (kumaranvpl) [reply openai_agent/send_email.py:1]: Okay. I will do it.

File: openai_agent/send_email.py (hunk 2 above)
Comment (kumaranvpl) [reply openai_agent/send_email.py]: support@captn.ai, It is not created yet though

File: openai_agent/send_email.py (hunk 1 above)
Comment (kumaranvpl) [reply openai_agent/send_email.py]: This is done

**Review by rjambrecic — APPROVED** [review]

**Pull request merged**